### PR TITLE
Use Go 1.11 modules

### LINKED ISSUE #202: Add support for vgo (go.mod file)
Would nice to have for anyone using vgo. Should be simple as there is already a Gopkg.toml file.

### PR DESCRIPTION
**Pull request by maxekman**
Including bumping of a few deps which now also have modules support (this gets rid of some `// indirect` lines):
- github.com/grpc-ecosystem/go-grpc-middleware
- github.com/grpc-ecosystem/go-grpc-prometheus
- github.com/rs/cors

Did not bump any other deps, upgrades can be listed with `go list -m -u all` from the root folder.

Fixes #202.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	github.com/sirupsen/logrus v1.0.4
+	github.com/spf13/pflag v1.0.0
+	github.com/stretchr/testify v1.1.4
+	golang.org/x/crypto v0.0.0-20180910181607-0e37d006457b // indirect
+	golang.org/x/net v0.0.0-20171207222016-dc871a5d77e2
+	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
+	golang.org/x/sys v0.0.0-20180909124046-d0be0721c37e // indirect
+	golang.org/x/text v0.3.0 // indirect
+	google.golang.org/genproto v0.0.0-20180831171423-11092d34479b // indirect
+	google.golang.org/grpc v1.14.0
```
Comment (johanbrandhorst) [inline go.mod]: I don't think this will work with `grpcwebproxy`. It was locked to `v1.8.0` in `Gopkg.toml`.

Comment (maxekman) [reply go.mod]: Strange, it did get the initial versions from Gopkg.toml. I only bumped the above mentioned (to get a cleaner go.mod file).

Comment (maxekman) [reply go.mod]: Should I also update the `grpcwebproxy` to work with 1.14.0 if that is the case?

**Comment by maxekman** [comment]
I set the version of grpc back to v1.8.0 (as it was in Gopkg.toml). Really strange that it did not use that from the beginning, all other versions got moved over..

**Comment by johanbrandhorst** [comment]
Nice work. We'd like to upgrade gRPC of course but I think it'd be more work than you think.

**Comment by johanbrandhorst** [comment]
Ah yes that PR sounds like lt'd allow us to upgrade.

**Comment by maxekman** [comment]
Perfect. Do I need to do anything to get TravisCI to run the check?

**Comment by johanbrandhorst** [comment]
I'm not quite sure what's going on here, it should start automatically :thinking:.

**Comment by johanbrandhorst** [comment]
This PR is not showing up on https://travis-ci.org/improbable-eng/grpc-web/pull_requests. The plot thickens...

**Comment by johanbrandhorst** [comment]
I've just merged a thing to master, could you try re-basing and pushing to see if it triggers a job?

**Comment by maxekman** [comment]
Done.

**Comment by johanbrandhorst** [comment]
Really not sure what's going on here, but lets give it a few hours and see if anything shakes out.

**Comment by maxekman** [comment]
It was like this from the moment I submitted the PR yesterday, not sure if more time helps.

**Comment by maxekman** [comment]
I think I know what it is, and I had a similar problem before. I have allowed access to travis-ci.COM not .ORG for my Github user. This caused me a lot of confusion when setting up github.com/looplab/eventhorizon as travis-ci.com and travis-ci.org have different API keys on Github. They recommend new projects to use the .com version not the .org version.

**Comment by maxekman** [comment]
My account was also flagged for "potential abuse" for some reason, I have now mailed them.

**Comment by maxekman** [comment]
screen shot 2018-09-12 at 11 13 58
There we have it!

**Comment by maxekman** [comment]
Found it under "requests", the button to the right with the +.

**Comment by johanbrandhorst** [comment]
Oh, interesting :thinking:. So what do we do about it?

**Comment by maxekman** [comment]
I contacted their support to remove the flagging, let’s see if that helps.

**Comment by maxekman** [comment]
Btw, they just merged https://github.com/mwitkow/grpc-proxy/pull/27, should be possible to update the proxy after they make a release.

**Comment by johanbrandhorst** [comment]
Good to see, I pulled some strings ;). I think we'll just track that with https://github.com/improbable-eng/grpc-web/issues/236. Feel free to help :)!

**Comment by johanbrandhorst** [comment]
Superceded by #557

**Pull request closed without merging**